modules.d/50drm/module-setup.sh

```diff
@@ -32,7 +32,7 @@ installkernel() {
     # loading of the driver if needed
     if [[ $hostonly ]]; then
         for i in /sys/bus/{pci/devices,platform/devices,virtio/devices,soc/devices/soc?}/*/modalias; do
-            [[ -e $i ]] || continue
+            [[ -e $i && -n $(<"$i") ]] || continue
```

Comment:
technically, the `-e` is not required, as the `for i in <wildcard>` will already return only existing things, and `-e` doesn't test the file for anything else.